### DISCUSSION TITLE
2nd type of turtle

### DISCUSSION THREAD

**Discussion by emmemd** [IntelligenceModding/AdvancedPeripherals#169] [Ideas]
So i was looking at the end automata and thought it is cool but getting a turtle to travel to each location first is a nightmare with the turtles speed. now i know traveling to an warping to an XYZ is a bit op (although it would be nice) I was wondering if it would be possible to have a 2nd "scout" computer that has limited functionality like in open computers. but was faster and could do things like mark warp locations for other computers.

**Comment by SirEdvin** [comment]
Oh, this defenitly should be considered, but need some thinking.

For what are you planning to use such a scout computer? In any case you will be limited by chunk loading logic (because most of servers hates moving chunk loading)

**Reply by emmemd** [reply]
the scout computer should be allowed to do light tasks such as use the environmental scanner to find biomes. it should probably be cheaper of course faster and be able to work with select peripherals. it would be nice if it could have a built in gps/built in pathfinding script (although that should come as a separate peripheral). i personally like to use this mod to annoy my friends so i would use a lighter faster turtle to move to their xyz then set a warp point for other turtles to use. allowing them to attack would be nice however it should be no more powerful than a punch/ doesnt accept swords as a utility. I would also use it to support/ refuel other turtles. to solve the chunk loading problem this turtle can be equipped with a chunk loader this theoretically should be one of the major uses for it. This is just a quick look at it and can almost certainly be used for much more

**Reply by emmemd** [reply]
for the moving chunk loader surely it would operate similar to a player. it may have performance problem but maybe put it under the items description. thanks for considering it
